src/patternfly/components/Hint/hint.scss

```diff
@@ -0,0 +1,85 @@
+.pf-c-hint {
+  --pf-c-hint--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-hint--BackgroundColor: var(--pf-global--palette--blue-50);
+  --pf-c-hint--BorderColor: var(--pf-global--palette--blue-100);
+  --pf-c-hint--BorderWidth: var(--pf-global--BorderWidth--sm);
```

Comment:
Should we use a `::before` pseudo for borders?

Reply:
I didn't add a pseudo because there's no hover/focus state on the hint

Reply:
but I could, in case it has a state in the future